stepheng323/Automart
# 166860172 write API tests 166860172

#### What does this PR do?
Write out a unit test for all endpoints using mocha and chai-http 
#### Description of Task to be completed?

- POST  /api/v1/auth/signup

- POST /api/v1/auth/signin

- POST /api/v1/car'

- POST /api/v1/order

- PATCH /api/v1/order/:id/price'

- PATCH /api/v1/car/:id/price'

- PATCH api/v1/car/:id/status

- GET api/v1/car/:id

- GET /api/v1/car

- GET /api/v1/car?status=available&min_price=​ XXXValue​ &max_price=​ XXXValue

- GET /api/v1/car?status=available

- DELETE /api/v1/car/:car_id

#### How should this be manually tested?

- it should be run against the code for the listed endpoints above

- setup a database on your local machine or online

- create a user abiodun

- Import the sql file in the test folder to your database 

- Cd into the directory and use npm test to launch test

#### What are the relevant pivotal tracker stories?
166860172
